src/common.c

```diff
@@ -2077,6 +2077,7 @@ void ofi_get_list_of_addr(const struct fi_provider *prov, const char *env_name,
 	for (ifa = ifaddrs; ifa != NULL; ifa = ifa->ifa_next) {
 		if (ifa->ifa_addr == NULL ||
 			!(ifa->ifa_flags & IFF_UP) ||
+			!(ifa->ifa_flags & IFF_RUNNING) ||
```

Comment:
IFF_RUNNING isn't defined on windows, resulting in a build error there.  At least IFF_UP and IFF_LOOPBACK are defined by windows, but I haven't figured out what header file defines them (guessing winsock2.h) somewhere.  The best option here may be to add this to include/windows/ifaddrs.h:
```
#ifndef IFF_RUNNING
#define IFF_RUNNING IFF_UP
#endif
```
It's a hack, but should work and is no worse off on windows than we are now.

Reply:
@shefty done, but will wait for testing and will re-push it since extra space at the end of the comment.